src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -592,11 +594,18 @@ public void selectDropdownByActualValue(WebElement element, String value) {
     }
 
     /**
-     * Returns the status message in the page. Returns "" if there is no
-     *         status message in the page.
+     * Returns a list of Texts of user status messages in the page.
+     * Returns a list containing "" if there is no status message in the page.
      */
-    public String getStatus() {
-        return statusMessage == null ? "" : statusMessage.getText();
+    public List<String> getTextsForAllUserStatusMessages() {
```

Comment:
Let's just stick to `getTextsForAllStatusMessagesToUser()` for consistency although verbose :(.

Let's just do this: `statusMessage.findElements(By.tagName("div"));`